### DISCUSSION TITLE
It is impossible to complete the academic reverification

### DISCUSSION THREAD

**Discussion by saliudev** [community/community#49348] [GitHub Education]
I received an email asking for academic reverification, which I totally understand why it is necessary. However, it is basically impossible to complete the verification process for me.

1. I can't upload a pdf or basically any files to the verification portal. The only thing it accepts is camera on your computer, which is ridiculous. What if I don't have a camera on my computer? 
2. It asks for a date for the ID. I am not sure which institution still print date on their ID. Anyone?
3. The picture I took was rejected, and no reason was provided. It only points me to a generic website that can not solve anything: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved

I love using Github in my classroom, but this road block process is just unnecessary, waste of my time and stupid.  Why can't I just simply upload a file or provide a link to institution profile to verify the status?

**Comment by samus-aran** [comment]
Hi @saliudev 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students (and faculty) everywhere to verify their academic status, while also ensuring that bad actors impersonating students (and faculty) don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the user applying and we thank you for being patient.

Here are some tips to help you resolve some of your rejection reasons:

It would be preferable that a user is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request users submit a live photo of their evidence rather than an upload. We understand that users may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

We do prefer a dated student/faculty ID but we understand not all institutes do - there are still many that do. So please if you have a Faculty ID card you can submit this as evidence. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top, followed by Identification card. 

Hope this helps. Let us know how you get on 👍

**Comment by Shifath472533** [comment]
true. the process is dumb
